readme.md

```diff
@@ -1,10 +1,10 @@
 # Bru Lang
 
-Bru is a simple markup language with json like semantics.
+Bru is a simple markup language with [JSON](https://json.org)-like semantics.
```

Comment:
This points people to the main syntactical definition of JSON, for which Bru will be leaning on the *meaning* of most supported primitive types. That is, a Bru `number` is the same as a JSON `number`.

Reply:
👍